src/main/webapp/script.js

```diff
@@ -31,3 +31,44 @@ async function checkLog() {
     logButton.innerText = "LOGIN";
   }
 }
+
+function showSettings(contentId) {
+  var settings = document.getElementsByClassName("marker-setting")[0];
+  document.getElementById("submit-button").onclick = function () {
+    updateMarkerSettings(contentId);
+  };
```

Comment:
I think that when clicking the settings button, the user should see the previous details before being able to edit them. But we will also solve this in a future PR.

Reply:
Oh yeah I thought about that too! I have three possible solutions to that:
1) send request to the servlet to obtain settings for chosen marker (easy but time consuming)
2) create invisible block with info about every marker when loading route (a bit harder in the terms of number of lines of code)
3) create separate settings section for each marker (super hard in the same terms and extremely messy)

I personally like the second option better, do you have any other thoughts about that?

Reply:
I also prefer number 2, but agree it should be done in a future PR.